packages/lexical-playground/src/plugins/DraggableBlockPlugin/index.tsx

```
@@ -18,7 +18,6 @@ import {
   DROP_COMMAND,
   LexicalEditor,
 } from 'lexical';
-import {getElementByKeyOrThrow} from 'lexical/src/LexicalUtils';
```

We could also just export it, I guess?

---

I noticed that in other plugins, editor.getElementByKey is used when you need to get the element by key. whether we export getElementByKeyOrThrow or not, it looks better to use editor.getElementByKey consistently here. What do you think?

---

I don't have a strong preference - it's mostly just a convenience method, but now you've gone from throwing an error in the case of a key lookup miss to just silently failing. If that's the behavior you want, then this is fine.

---

Thank you for the detailed explanation here. <3